diff --git a/crates/atuin-client/src/database.rs b/crates/atuin-client/src/database.rs
index d01dadb4..770ebb29 100644
--- a/crates/atuin-client/src/database.rs
+++ b/crates/atuin-client/src/database.rs
@@ -140,7 +140,7 @@ impl Sqlite {
         let opts = SqliteConnectOptions::from_str(path.as_os_str().to_str().unwrap())?
             .journal_mode(SqliteJournalMode::Wal)
             .optimize_on_close(true, None)
-            .synchronous(SqliteSynchronous::Normal)
+            .synchronous(SqliteSynchronous::Off)
             .with_regexp()
             .create_if_missing(true);
 
